include/deal.II/lac/scalapack.h

```diff
@@ -695,6 +707,12 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
    */
   mutable std::vector<int> iwork;
 
+  /**
+   * Integer array holding pivoting information required
+   * by ScaLAPACK's matrix factorization routines.
+   */
+  mutable std::vector<int> ipiv;
```

Comment:
since neither `compute_lu_factorization()` nor `invert` is `const`, you can drop `mutable`